pydoctor/test/test_packages.py

```diff
@@ -70,3 +70,10 @@ def test_allgames() -> None:
     assert moved.parentMod is mod2
     assert moved.parentMod.source_path is not None
     assert moved.parentMod.source_path.parts[-2:] == ('allgames', 'mod2.py')
+
+def test_cyclic_imports() -> None:
+    system = processPackage('cyclic_imports')
```

Comment:
I would like to see a docstring with the target / scope / purpose of this test :)

Reply:
I'll add it.